### PR TITLE
chore: fix the publishing of the monitor-demo packages

### PR DESCRIPTION
**Pull request by oleorhagen**
The logic was not ready for multiple similarly named packages.

Changelog: None
Signed-off-by: Ole Petter <ole.orhagen@northern.tech>

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -386,16 +390,20 @@ publish:s3:mender-monitor:automatic:
     - *publish_helper_functions
   script:
     - echo "Publishing mender-gateway version ${MENDER_GATEWAY_VERSION} to s3://${S3_BUCKET_NAME_PRIVATE}/${S3_BUCKET_SUBPATH_GATEWAY}/${MENDER_GATEWAY_VERSION}/"
-    - aws s3 cp output/commercial/mender-gateway*.deb s3://${S3_BUCKET_NAME_PRIVATE}/${S3_BUCKET_SUBPATH_GATEWAY}/${MENDER_GATEWAY_VERSION}/
+    - aws s3 cp output/commercial/ s3://${S3_BUCKET_NAME_PRIVATE}/${S3_BUCKET_SUBPATH_GATEWAY}/${MENDER_GATEWAY_VERSION}/ --recursive --exclude "*" --include "mender-gateway*.deb"
```
Comment (lluiscampos) [inline .gitlab-ci.yml:393]: How this differs from the previous command?

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -386,16 +390,20 @@ publish:s3:mender-monitor:automatic:
...
     # Make copies in known destinations to be consumed by get.mender.io script:
     # * For "experimental" channel, we make a copy of the newest master package in the master directory
     # * For "stable" channel, we make a copy in a separate "latest" subdirectory of the latest tagged version
-    # This needs to be reworked before Mender 3.2, see MEN-5029 for more details.
+    # This can to be reworked for better maintainability, see MEN-5029 for more details.
```
Comment (lluiscampos) [inline .gitlab-ci.yml:397]: Thanks

**Review by lluiscampos — APPROVED** [review]

File: .gitlab-ci.yml (hunk 1 above)
Comment (oleorhagen) [reply .gitlab-ci.yml:393]: The awscli command is not too happy about multiple packages it seems:

```
cp
          <LocalPath> <S3Uri> or <S3Uri> <LocalPath> or <S3Uri> <S3Uri>
```

Comment (lluiscampos) [reply .gitlab-ci.yml:393]: Thanks :+1: Rings a bell, actually.

**Pull request merged**